typescriptFixme.ts

```diff
@@ -1,115 +1,143 @@
 // add the type annotation to explicitly specify the type of the variable:
-var isDone = false;
-var lines = 42;
-var name = "Anders";
+var isDone: boolean = false;
+var lines: number = 42;
+// let name: string = "Anders";
 
 // annotate the type as when it's not known
-let unknownTypeVariable;
+let unknownTypeVariable: any;
 
 //annotate this array
-var list = [1, 2, 3];
+var list: number[] = [1, 2, 3];
 
 // implement color in typescript
-var Color = { Red: 0, Green: 1, Blue: 2 };
-var c = Color.Green;
+enum Color {
+  Red = 0,
+  Green = 1,
+  Blue = 2,
+}
+var c: Color = Color.Green;
 console.log(Object.keys(Color)[c]); // "Green"
 
-function littleHorribleAlert() {
+function littleHorribleAlert(): void {
   alert("I am a little annoying box!!");
 }
 
-var f1 = function (i) {
+var f1 = function (i: number): number {
   return i * i;
 };
 // as f1 just don't specify the return type here
-var f2 = function (i) {
+var f2 = function (i: number) {
   return i * i;
 };
 
-var f3 = (i) => {
+var f3 = (i: number): number => {
   return i * i;
 };
 
 //change f4 to accept a string or a number as a parameter
-function f4(i) {
+function f4(i: string | number): void {
   console.log("The value was " + i);
 }
 
 // Annotate the person objects with an interface. Some properties are optional
-var person1 = { name: "Bobby", move: function () {} };
-var person2 = { name: "Bobby", age: 42, move: function () {} };
+
+interface Person {
+  name: string;
+  age?: number;
+  move: () => void;
+}
+
+var person1: Person = { name: "Bobby", move: function () {} };
+var person2: Person = { name: "Bobby", age: 42, move: function () {} };
 
 // Annotate mySearch to describe a function that accepts two string parameters
-var mySearch = function (src, sub) {
+var mySearch = function (src: string, sub: string): boolean {
```

Comment:
not this. I want you to annotate mySearch. see https://www.typescriptlang.org/docs/handbook/2/functions.html#call-signatures